.gitignore

```diff
@@ -1,2 +1,4 @@
+stockbot.properties
 env
+venv
```

Comment:
What is this?

Reply:
When I create my virtual environment in Python, it is a convention to call it venv.